start.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cd $BOT_DIR
+
+logstamp=$(date +%Y%m%d-%H%M%S)
+mkdir -p logs/
```

Comment:
Might be good to gitignore this, for convenience

Reply:
Done. :+1: